worlds/shivers/data/regions.json

```
@@ -140,6 +149,29 @@
 		["To Puzzle Room Marbles From Skull Dial Bridge", "Puzzle Room Marbles"],
 		["To Skull Dial Bridge From Puzzle Room Marbles", "Skull Dial Bridge"],
 		["To Skull Dial Bridge From Slide Room", "Skull Dial Bridge"],
-		["To Slide Room", "Slide Room"]
+		["To Slide Room", "Slide Room"],
+		["To Wax Capture From Library", "Wax Capture"],
+		["To Wax Capture From Shaman", "Wax Capture"],
+		["To Wax Capture From Anansi", "Wax Capture"],
+		["To Water Capture From Lobby", "Water Capture"],
+		["To Water Capture From Janitor Closet", "Water Capture"],
+		["To Ash Capture From Office", "Ash Capture"],
+		["To Ash Capture From Burial", "Ash Capture"],
+		["To Oil Capture From Prehistoric", "Oil Capture"],
+		["To Oil Capture From Tar River", "Oil Capture"],
+		["To Cloth Capture From Egypt", "Cloth Capture"],
+		["To Cloth Capture From Burial", "Cloth Capture"],
+		["To Cloth Capture From Janitor Closet", "Cloth Capture"],
+		["To Wood Capture From Workshop", "Wood Capture"],
+		["To Wood Capture From Gods Room", "Wood Capture"],
+		["To Wood Capture From Anansi", "Wood Capture"],
+		["To Wood Capture From Blue Maze", "Wood Capture"],
+		["To Crystal Capture From Lobby", "Crystal Capture"],
+		["To Crystal Capture From Ocean", "Crystal Capture"],
+		["To Sand Capture From Greenhouse", "Sand Capture"],
+		["To Sand Capture From Ocean", "Sand Capture"],
+		["To Metal Capture From Bedroom", "Metal Capture"],
+		["To Metal Capture From Projector Room", "Metal Capture"],
+		["To Metal Capture From Prehistoric", "Metal Capture"]
 	]
 }
```

Newline at end of file
```suggestion
}

```

---

Added